src/logo-system/logo-elements/index.md

```diff
@@ -11,13 +11,10 @@ As our primary identifier, the GOV.UK wordmark should be used in all application
 
 [The exception to this rule is the GOV.UK website](/logo-system/web/).
 
-{% sectionHighlight %}
-
-<div class="app-section-highlight__wrapper--logo">
+{% sectionHighlight { classes: 'app-section-highlight__wrapper--logo govuk-!-margin-top-6 govuk-!-margin-bottom-6' } %}
```

Comment:
In a separate PR I'm gonna shuffle these classes around into a modifier on `app-section-highlight` to save a bit of space. Doesn't need to be done right now.